packages/server/graphql/schema.js

```diff
@@ -3,31 +3,75 @@ const {
   GraphQLObjectType,
   GraphQLID,
   GraphQLList,
-  GraphQLNonNull
+  GraphQLNonNull,
+  GraphQLString
 } = require("graphql");
-const fromGlobalId = require("graphql-relay").fromGlobalId;
-const productGraphQLType = require("./productType");
-const Product = require("../models/Product");
+const { fromGlobalId, mutationWithClientMutationId } = require("graphql-relay");
+const eventGraphQLType = require("./eventType");
+const Event = require("../models/Event");
 
 const Query = new GraphQLObjectType({
   name: "Query",
   fields: {
-    product: {
-      type: productGraphQLType,
+    event: {
+      type: eventGraphQLType,
       args: { id: { type: GraphQLNonNull(GraphQLID) } },
       resolve(parent, args) {
-        return Product.findById(fromGlobalId(args.id).id);
+        return Event.findById(fromGlobalId(args.id).id);
       }
     },
-    products: {
-      type: GraphQLList(productGraphQLType),
+    events: {
+      type: GraphQLList(eventGraphQLType),
```

Comment:
this is good, but won't scale
we should use connection for pagination:
https://github.com/facebook/relay/blob/a1e9a4b395f3eed6dfe58703a5fd02d0765b0672/packages/relay-runtime/handlers/connection/RelayConnectionInterface.js#L19-L28

Reply:
I'll add pagination by the end of the project. It seems a little complex x_x